deps/build.jl

```diff
@@ -1,4 +1,7 @@
-@static if is_apple() SYSTEM_BREW = "/usr/local/Homebrew/bin/brew" end
+@static if is_apple()
+    using Homebrew    # using Homebrew earlier to avoid doing eval in find_llvm
```

Comment:
IIRC a very large part of the goal of that section was to *avoid* using Homebrew if at all possible because `using Homebrew` can be very expensive.

Reply:
(but it has been a while since I looked in detail, just skimming open PRs here)